src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -771,6 +774,218 @@ allocate(const std::vector<int>& inteHead)
     };
 }
 
+template <typename SGProp, class SGrpArray>
+void assignGroupGasInjectionTargets(const Opm::Group&        group,
+                                    const Opm::SummaryState& sumState,
+                                    SGProp&&                 sgprop,
+                                    SGrpArray&               sGrp)
+{
+    using Ix = ::Opm::RestartIO::Helpers::VectorItems::SGroup::inj_index;
+    using M  = ::Opm::UnitSystem::measure;
+
+    const auto& prop = group.injectionProperties(Opm::Phase::GAS);
+    const auto  cntl = group.injectionControls(Opm::Phase::GAS, sumState);
```

Comment:
> Inconsistent. Use `const auto&`?

I decided against it since [`Group::injectionControls()`](https://github.com/OPM/opm-common/blob/2aff28dd575327af87b464ae174969ed7948f322/opm/input/eclipse/Schedule/Group/Group.hpp#L288) returns a value instead of a reference to `const`.  While lifetime extension would certainly allow using `const auto&` here, the downside to lifetime extension is that the compiler effectively must implement it through a free store allocation as
```C++
Group::InjectionControls* _cntl = new Group::InjectionControls{ group.injectionControls(...) };
const Group::InjectionContorls& cntl = *_cntl;
use(cntl);
delete _cntl;
```